src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,21 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import './TodoInfo.scss';
+import classnames from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => {
+  const { title, completed, user } = todo;
+
+  return (
+    <article
+      className={classnames({
+        TodoInfo: true,
+        'TodoInfo--completed': completed,
+      })}
+    >
+      <h2 className="TodoInfo__title">{title}</h2>
+
+      {user && <UserInfo user={user} />}
+
+    </article>
```

Comment:
```suggestion
      {user && <UserInfo user={user} />}
    </article>
```
Don't leave empty lines between parent and child elements